src/lib/api/core/base.ts

```diff
@@ -161,7 +165,7 @@ export abstract class BaseFxDirective implements OnDestroy, OnChanges {
   protected _applyStyleToElement(style: StyleDefinition,
                                  value?: string | number,
                                  nativeElement?: any) {
-    let element = nativeElement || this._elementRef.nativeElement;
+    let element = nativeElement || this.nativeElement;
```

Comment:
Same approach as https://github.com/angular/flex-layout/pull/396/files#diff-172881d6558fa5a20f715647057c40c2R137.

Reply:
See comment above regarding why `let element = nativeElement || this.nativeElement` is better than a default parameter value.